### DISCUSSION TITLE
Looking for Old xdrip version with 9 minutes rhythm

### DISCUSSION THREAD

**Discussion by PeterBaetschi** [NightscoutFoundation/xDrip#3037] [Q&A]
Dear friends

My Samsung Galxy 20fe 5G worked very well with Dexcom G6 using my Galaxy Watch 5pro as wear collector. That means my Galaxy Watch 5pro worked stand alone and read the Dexcom G6
My Samsung Galaxy 20fe 5G with xdrip works very well with dexcom G7/one. But when I force the Samsmg Galaxy phone to use the wear collector, it doesn't work.
-> for reading the Dexom G6 with the Watch as wear collector, I used an old xdrip version, that reads out every 9 minutes.

Now I look for this old xdrip apk with the 9 minutes rhythm. But I can not find this apk anymore.

Can someone send my a link to this old xdrip apk?

Thank you very much for your help!

**Comment by PhilTU** [comment]
When was this from?

**Reply by PeterBaetschi** [reply]
I loaded it down in about June this year

**Reply by PeterBaetschi** [reply]
In this 9' Version, the Watch sent i every p minutes a request to the G6 (i saw it like a push on the watch). Now, when I force the Habdy to use the watch collector, the Watch bilds up a connection, but does not read out the gluco number.

**Reply by PeterBaetschi** [reply]
![Uploading Screenshot_20230827_155217_xDrip+.jpg…]()

**Comment by PhilTU** [comment]
> I loaded it down in about June this year

you can go back forever at https://github.com/NightscoutFoundation/xDrip/releases and pick up apk for older releases from "assets" section.

**Reply by PeterBaetschi** [reply]
Thank you very much - I load it down in June, but I guess the change from 9' rhythm to 5' was in 2020/2021?

**Reply by PhilTU** [reply]
You can go back to 2016. I didn't know when this thing changed Peter, there's a changelog at https://navid200.github.io/xDrip/docs/ReleaseNotes.html 

https://github.com/NightscoutFoundation/xDrip/releases?page=20 is July 21, just edit the page number (larger number = earlier releases)

**Reply by PeterBaetschi** [reply]
Thank you very much for your help - meanwhile I think, it's maybe a problem between the android  version of my phone and the dexcom G7.

Do you know, which ist the best system to use the Galaxy watch 5pro as collector standalone? I try to get more information about the freestyle libre3.
But I would only change anyway from Dexcom, if I'm sure it's solution for my need 😉👍

**Reply by PeterBaetschi** [reply]
With xdrip!

**Reply by PhilTU** [reply]
the xDrip WearOS component can do standalone with a limited set of sesnsors. Libre 3 is not one of them. https://xdrip.readthedocs.io/en/latest/smartwatch/usewear/#standalone-collector-mode suggests Dexcom might be.
